python/openinference-semantic-conventions/src/openinference/semconv/trace/__init__.py

```diff
@@ -124,7 +124,8 @@ class MessageAttributes:
     """
     MESSAGE_CONTENT = "message.content"
     """
-    The content of the message to the llm
+    The content of the message to the llm, can be a string or an array of
```

Comment:
I think we leave this one alone and add a plural that supports lists


This means an image with text would look like

```

message.contents.0.message_content.type: text
message.contents.0.message_content.text: text

message.contents.1.message_content.type: image
message.contents.0.message_content.image.image.url: "https://mydog.png"